src/docs/asciidoc/getting_started/index.adoc

```diff
@@ -775,6 +775,106 @@ If you look at the *message.handlebars* file in the bundle, you will see how the
+
+```
+
+The build-in template supposes the message is stored in the card in field `data.message`.
+
+==== User card template
+
+If you want to create a user card for a simple message, just add in your handlebar file :
+
+```
+<opfab-message-usercard> </opfab-message-usercard>
+
+```
+
+The message will be stored in the field `data.message` of the card
+
+You can package the bundle and send it to the server with the provided scripts :
+
+----
+./packageBundle.sh | ./sendBundle.sh
+----
+
+Now if you reload the page, you should be able to create a card using build-in template. Don't forget to send it to an
+entity you're a part of to receive it. For instance *Control Center FR North*.
+
+image::example7_card_send.png[example 7 screen shot #1]
+
+
+You can change the text header by providing the message-header attribute:
+
+```
+<opfab-message-card  message-header="a new header"> </opfab-message-card>
+```
+
+By using attributes you can set some parameters regarding recipients, see
+ifdef::single-page-doc[<<'build-in_templates_common_usercard_attributes,common attributes for user cards'>>]
+ifndef::single-page-doc[<</documentation/current/reference_doc/index.adoc#build-in_templates_common_usercard_attributes, common attributes for user cards>>]
+
+You can see the updated UI using this command line and refreshing the page to update the bundle.
+
+----
+./packageBundle_updated.sh | ./sendBundle.sh
+----
+
+image::example7_card_send_updated.png[example 7 screen shot #2]
+
+=== Example 8: Question card using "build-in" templates
+
+Instead of coding your own templates for question cards or user cards, you can use opfab build-in templates if it suits your needs.
+
+==== Card template
+
+If you want to show a question and see user responses, you can use the question build-in template, to do that just put in your handlebar file:
+```
+<opfab-question-card> </opfab-question-card>
+
+```
+
+The build-in template supposes the question is stored in the card in the field `data.question`.
+
+==== User card template
+
+If you want to create a user card for a simple question, just add in your handlebar file :
+
+```
+<opfab-question-usercard> </opfab-question-usercard>
+
+```
+The question will be stored in the field `data.question` of the card
+
+By using attributes you can set some parameters regarding recipients, see
+ifdef::single-page-doc[<<'build-in_templates_common_usercard_attributes,common attributes for user cards'>>]
+ifndef::single-page-doc[<</documentation/current/reference_doc/index.adoc#build-in_templates_common_usercard_attributes, common attributes for user cards>>]
+
+Use the provided script to add rights to send question card :
+----
+./setupPerimeter.sh perimeter.json
+----
+
+You can package the bundle and send it to the server with the provided scripts :
+
+----
+./packageBundle.sh | ./sendBundle.sh
+----
+
+Now if you reload the page, you should be able to create a question card using build-in template.
```

Comment:
Add a scenario : 
Operator 1 send question 
Operator 2 connect and respond to question
Operator 1 connect and see response

Reply:
Done.